src/catcollar/mod.rs

```diff
@@ -459,11 +458,11 @@ impl CatcollarLibOS {
 // Standalone Functions
 //==============================================================================
 
-/// Parses a [Ipv4Endpoint] into a [SockaddrStorage].
-fn parse_addr(endpoint: Ipv4Endpoint) -> SockaddrStorage {
-    let addr: Ipv4Addr = endpoint.get_address();
-    let port: u16 = endpoint.get_port().into();
-    let ipv4: SocketAddrV4 = SocketAddrV4::new(addr, port);
+/// Parses a [SocketAddrV4] into a [SockaddrStorage].
+fn parse_addr(endpoint: SocketAddrV4) -> SockaddrStorage {
+    let addr: &Ipv4Addr = endpoint.ip();
+    let port: u16 = endpoint.port().into();
```

Comment:
Can't we remove the ".into()" now?  I believe port for a SocketAddr* is defined as returning a u16.

Reply:
Solved the `into()` call.